Sources/PackageModel/SwiftSDKs/SwiftSDKConfigurationStore.swift

```diff
@@ -202,6 +202,7 @@ public final class SwiftSDKConfigurationStore {
             }
 
             if showConfiguration {
+                print("\nSDK: \(targetTriple)")
```

Comment:
These are not SDKs, but Swift SDKs. These are two different concepts that have different implementation and different set of options etc. We should make the distinction clear to end users.

```suggestion
                print("\nSwift SDK: \(targetTriple)")
```